pkg/extensions/search/utils/util.go

```diff
@@ -0,0 +1,532 @@
+package utils
```

Comment:
This seems like it could be better than "utils". "types", at least? "cve-info"?

Reply:
Yes, I will change this